### ISSUE TITLE
contributes_to: activity for myosin regulatory chain?

### ISSUE THREAD

**Issue by pfey03** [geneontology/go-annotation#2994]
@bmeldal and @hattrill 

I have this contributes_to
P13833  mlcR  contributes_to  GO:0030898 actin-dependent ATPase activity  IDA

Looking at the GO, I find the BP
GO:1904621  regulation of actin-dependent ATPase activity

However, this is a BP and not MF. So I will loose the MF and gain another BP, and the well known regulatory chain has then no MF

Do we need an MF - instead of BP??
actin-dependent ATPase regulator activity

Thanks!

**Comment by bmeldal** [comment]
Yes, NTR - unless the direct (more generic) parent* of your proposed term would be fine, too.

* I don't know what the parent would be ;-)

**Comment by pfey03** [comment]
Yes @bmeldal , terms under 
GO:0030234 enzyme regulator activity
are like this. There are already a lot of regulator activity terms.

**Comment by pfey03** [comment]
Ah and now understood more what you meant
GO:0060590 ATPase regulator activity

This term is there. Looses the typical actin part for an actin activated ATPase (myosin), so I could use this and then would still add the BP term
@bmeldal do you think that's a good solution?

**Comment by bmeldal** [comment]
Might be enough :)

Depend how specific to the actin you need it to be. GO editors would love it if your term already exists ;-)

**Comment by pfey03** [comment]
yeah, maybe leave it as this to be able to move on. I find it pretty messy, which enzyme regulators have an activity (MF) and which don't. There seems no consistency. And once the MF is there, there is no equivalent BP. If I had less on my plate I could look into that and try to make sense... Thanks @bmeldal

**Comment by hattrill** [comment]
Hi @pfey03 - I had a look at our MLCs
Their desription in our gene groups is "Myosin light chains (MLC) bind myosin heavy chains via an IQ motif in the neck region. They are involved in myosin motor-based movement along actin filaments, regulating ATP hydrolysis or providing a structural function...."
The two MF terms that we have used are:
myosin heavy chain binding ; GO:0032036 and enzyme regulator activity ; GO:0030234

**Comment by pfey03** [comment]
Thanks @hattrill I do have a lot of annotations for the dicty myosin II it's been so much studied. Looking into some old papers I'm fleshing all annotations out with todays annotation knowledge. These where annotated in 2002/3.

**Comment by ValWood** [comment]
ca this close? assigning to @pfey03 to review and decide.

**Comment by ValWood** [comment]
Closing, this discussion seems to be complete?

**Comment by pfey03** [comment]
I will review next week if any are left

**Comment by pfey03** [comment]
I'm done it can be closed
